Doc/library/pydoc.rst

```diff
@@ -65,25 +65,25 @@ manner similar to the Unix :program:`man` command.  The synopsis line of a
 module is the first line of its documentation string.
 
 You can also use :program:`pydoc` to start an HTTP server on the local machine
-that will serve documentation to visiting web browsers.  :program:`pydoc -p 1234`
+that will serve documentation to visiting web browsers. :program:`<python> -m pydoc -p 1234`
 will start a HTTP server on port 1234, allowing you to browse the
 documentation at ``http://localhost:1234/`` in your preferred web browser.
 Specifying ``0`` as the port number will select an arbitrary unused port.
 
-:program:`pydoc -n <hostname>` will start the server listening at the given
+:program:`<python> -m pydoc -n <hostname>` will start the server listening at the given
 hostname.  By default the hostname is 'localhost' but if you want the server to
 be reached from other machines, you may want to change the host name that the
 server responds to.  During development this is especially useful if you want
 to run pydoc from within a container.
 
-:program:`pydoc -b` will start the server and additionally open a web
+:program:`<python> -m pydoc -b` will start the server and additionally open a web
```

Comment:
I think we can keep these.

Reply:
updated the doc

Reply:
Great! You can click "resolve conversation" after you feel that the review comment has been addressed.